lib/device.js

```diff
@@ -251,6 +256,169 @@ const util = require('util'),
                 }
                 return returnTxt;
             }
+        },
+        /**
+         * get screen capture of the given device
+         * @property options {String} device, display, outputPath
+         */
+        captureScreen: function(options, next) {
+            log.info("device#capture#captureScreen()");
+            if (typeof next !== 'function') {
+                throw new Error('Missing completion callback (next=' + util.inspect(next) + ')');
+            }
+            options = options || {};
+            async.series([
+                _makeSession,
+                _makeCaptureOption,
+                _captureFile,
+                _copyToLocal
+            ],  function(err, results) {
+                log.verbose("device#capture()", "err: ", err, "results:", results);
+                const resultTxt = "Create " + chalk.green(options.captureFileName) + " to " + options.destinationPath +"\nSuccess";
+
+                // clean up /tmp/capture directory in target device
+                if (options.targetCaptureDir) {
```

Comment:
Doesn't check if the actual file exists, is it correct to delete it unconditionally if it is in option?

Reply:
I had a talk it. I changed the value to createdTmpDir.
Done.